src/elements/ts-dialogs/ts-loading.html

```diff
@@ -62,6 +62,9 @@
             color: var(--accent-color);
             font-size: 90%;
             cursor: pointer;
+            max-width: 90%;
+            overflow-wrap: break-word;
+            margin: auto;
```

Comment:
better looking url (not overflowing)

![image](https://user-images.githubusercontent.com/34975907/115440537-d77bef00-a1dd-11eb-8219-b74999d49bea.png)